lib/src/webdav/client.dart

```diff
@@ -51,7 +72,8 @@ class WebDavClient {
 
   /// returns the WebDAV capabilities of the server
   Future<WebDavStatus> status() async {
-    final response = await _network.send('OPTIONS', _getUrl('/'), [200]);
+    final response =
+        await _network.send('OPTIONS', '$_baseUrl/remote.php/dav', [200]);
```

Comment:
Can we move the DAV-Url into a `final _davUrl` which gets assigned during construction?